GliaWidgets/Glia.swift

```diff
@@ -223,4 +280,58 @@ public class Glia {
             print("DB has not been removed due to: '\(error)'.")
         }
     }
+
+    /// Fetch current Visitor's information.
+    ///
+    /// The information provided by this endpoint is available to all the Operators observing or interacting with the
+    /// Visitor. This means that this endpoint can be used to provide additional context about the Visitor to the
+    /// Operators.
+    ///
+    /// - Parameters:
+    ///   - completion: A callback that will return the update result or `SalemoveError`
+    ///
+    /// If the request is unsuccessful for any reason then the completion will have an Error.
+    /// The Error may have one of the following causes:
+    ///
+    /// - `GeneralError.internalError`
+    /// - `GeneralError.networkError`
+    /// - `ConfigurationError.invalidSite`
+    /// - `ConfigurationError.invalidEnvironment`
+    /// - `ConfigurationError.invalidAppToken`
+    /// - `ConfigurationError.invalidApiToken`
```

Comment:
Applicable for both interfaces:
1. Since these errors are part of Salemove SDK, we need to add Salemove namespace to avoid misunderstanding with integrators
```suggestion
    /// - `Salemove.GeneralError.internalError`
    /// - `Salemove.GeneralError.networkError`
    /// - `Salemove.ConfigurationError.invalidSite`
    /// - `Salemove.ConfigurationError.invalidEnvironment`
    /// - `Salemove.ConfigurationError.invalidAppToken`
    /// - `Salemove.ConfigurationError.invalidApiToken`
```

2. To be more consistent, I'd like to propose to handle case if `interactor` is not initialized yet. Does it make sense?
```swift
public func fetchVisitorInfo(completion: @escaping (Result<Salemove.VisitorInfo, Error>) -> Void) {
    guard let interactor = self.interactor else { 
        return completion(GliaError.sdkIsNotConfigured)
    }
    Salemove.sharedInstance.fetchVisitorInfo(completion)
}
```

Reply:
Totally makes sense 4e55cb9419abeedae735bfd35de63fdc889a57eb.